lib/Doctrine/Common/DataFixtures/FixtureInterface.php

```diff
@@ -4,7 +4,8 @@
 
 namespace Doctrine\Common\DataFixtures;
 
-use Doctrine\Common\Persistence\ObjectManager;
+use Doctrine\Common\Persistence\ObjectManager as DeprecatedManager;
+use Doctrine\Persistence\ObjectManager;
```

Comment:
These are not 2 different classes, one is actually an alias to the other one, so there is no issue with using the new name in the signature because it's the same thing